diff --git a/configs/common/Caches.py b/configs/common/Caches.py
index f1b1d1e82..be757cab8 100644
--- a/configs/common/Caches.py
+++ b/configs/common/Caches.py
@@ -51,9 +51,9 @@ from m5.objects import *
 
 class L1Cache(Cache):
     assoc = 8
-    tag_latency = 0
-    data_latency = 0
-    response_latency = 0
+    tag_latency = 2
+    data_latency = 2
+    response_latency = 2
     mshrs = 4
     tgts_per_mshr = 20
 
@@ -67,9 +67,9 @@ class L1_DCache(L1Cache):
 
 class L2Cache(Cache):
     assoc = 16
-    tag_latency = 3
-    data_latency = 3
-    response_latency = 3
+    tag_latency = 6
+    data_latency = 6
+    response_latency = 6
     mshrs = 20
     tgts_per_mshr = 12
     write_buffers = 8
diff --git a/src/arch/x86/X86TLB.py b/src/arch/x86/X86TLB.py
index 8cfb6358a..bb35526cf 100644
--- a/src/arch/x86/X86TLB.py
+++ b/src/arch/x86/X86TLB.py
@@ -52,7 +52,7 @@ class X86TLB(BaseTLB):
     type = 'X86TLB'
     cxx_class = 'X86ISA::TLB'
     cxx_header = 'arch/x86/tlb.hh'
-    size = Param.Unsigned(1024, "TLB size")
+    size = Param.Unsigned(64, "TLB size")
     system = Param.System(Parent.any, "system object")
     walker = Param.X86PagetableWalker(\
             X86PagetableWalker(), "page table walker")
