diff --git a/configs/common/Caches.py b/configs/common/Caches.py
index f1b1d1e82..be757cab8 100644
--- a/configs/common/Caches.py
+++ b/configs/common/Caches.py
@@ -51,9 +51,9 @@ from m5.objects import *
 
 class L1Cache(Cache):
     assoc = 8
-    tag_latency = 0
-    data_latency = 0
-    response_latency = 0
+    tag_latency = 2
+    data_latency = 2
+    response_latency = 2
     mshrs = 4
     tgts_per_mshr = 20
 
@@ -67,9 +67,9 @@ class L1_DCache(L1Cache):
 
 class L2Cache(Cache):
     assoc = 16
-    tag_latency = 3
-    data_latency = 3
-    response_latency = 3
+    tag_latency = 6
+    data_latency = 6
+    response_latency = 6
     mshrs = 20
     tgts_per_mshr = 12
     write_buffers = 8
diff --git a/configs/simbricks/simbricks.py b/configs/simbricks/simbricks.py
index 58890ff3a..6478343b6 100644
--- a/configs/simbricks/simbricks.py
+++ b/configs/simbricks/simbricks.py
@@ -396,6 +396,8 @@ def build_system(np):
     # For now, assign all the CPUs to the same clock domain
     sys.cpu = [TestCPUClass(clk_domain=sys.cpu_clk_domain, cpu_id=i)
                     for i in range(np)]
+    sys.cpu[0].dtb.size = 64
+    sys.cpu[0].itb.size = 64
 
     if ObjectList.is_kvm_cpu(TestCPUClass) or \
         ObjectList.is_kvm_cpu(FutureClass):
